something.yml

```diff
@@ -0,0 +1,29 @@
+openapi: "3.0.0"
```

Comment:
to remove pls ^^

Reply:
Au secours ><
Should be ok now, for real!